<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); font-size: 28pt;
</string>
    </property>
    <property name="text">
     <string>Registrate!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 121, 0);
font-size: 14pt;</string>
    </property>
    <property name="text">
     <string>Correo Electronico:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font-size:17pt;
background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="contrasenia">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(46, 52, 54);
border-color:rgb(255, 255, 255);
font-size:17pt;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 121, 0);
font-size: 14pt;</string>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32, 74, 135);</string>
    </property>
    <property name="text">
     <string>Registrate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 121, 0);
font-size: 14pt;</string>
    </property>
    <property name="text">
     <string>Confirme contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="contrasenia_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(46, 52, 54);
border-color:rgb(255, 255, 255);
font-size:17pt;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(164, 0, 0);</string>
    </property>
    <property name="text">
     <string>Correo invalido</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 121, 0);
font-size: 14pt;</string>
    </property>
    <property name="text">
     <string>Nombres:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombres">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font-size:17pt;
background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="apellidos">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font-size:17pt;
background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 121, 0);
font-size: 14pt;</string>
    </property>
    <property name="text">
     <string>Apellidos:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font-size:17pt;
background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 121, 0);
font-size: 14pt;</string>
    </property>
    <property name="text">
     <string>Edad:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genero">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font-size:17pt;
background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 121, 0);
font-size: 14pt;</string>
    </property>
    <property name="text">
     <string>Genero:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 121, 0);
font-size: 14pt;</string>
    </property>
    <property name="text">
     <string>Nombre de Usuario:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font-size:17pt;
background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
